tailwind.config.js

```diff
@@ -151,7 +152,6 @@ module.exports = {
             orange: '#F9AA4B',
             yellow: {
                 DEFAULT: '#FAAD14',
```

Comment:
yellow is same as warning-300

Reply:
Removing yellow DEFAULT and replacing all of its instances with warning-500